### DISCUSSION TITLE
Welcome to pie_chart Discussions!

### DISCUSSION THREAD

**Discussion by apgapg** [apgapg/pie_chart#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iririya** [comment]
Hi! Just wondering is there any possibility to position legend with two constraints, not only one (like bottom-left, not only bottom)?

**Reply by apgapg** [reply]
Not really as of now. It will be difficult to achieve that. You can try fork and do the same

**Comment by harinandhan15** [comment]
Hi!

This package is running only with canvaskit render. Is there any way to run it in HTML render. When i take build for html render ui of pie chart is broken.

**Reply by apgapg** [reply]
No this has not been upgraded. You can fork and make necessary changes and would be happy if we receive a PR
